docs/source/tutorials/configuration.ipynb

```diff
@@ -1,20 +1,452 @@
 {
```

Comment:
<p>There are references to RADICAL SAGA, shouln't we replace it with psi/j</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>